src/internal/observable/ConnectableObservable.ts

```diff
@@ -37,6 +37,7 @@ export class ConnectableObservable<T> extends Observable<T> {
     }
   }
 
+  /** @internal */
```

Comment:
I tagged `_subscribe` as internal 'cause it was tagged in many other places and this was done as part of a search/replace thing. I've ignored the other protected members in this class 'cause the entire class is deprecated.